tests/test_parse.py

```diff
@@ -11,11 +11,11 @@
 @dataclass
 class Config:
     a: int
-    g: bool = field(metadata={"as_flags": True})
     b: str = "abc"
     c: float = 0.1
     d: bool = False
-    f: bool = field(default=True)
+    f: bool = field(default=True, metadata={"as_flags": False})
```

Comment:
not sure if it makes sense for this use case, but before the g variable didn't have a default value. Is that not supported anymore? 
Should the default be False when nothing is set, but `"as_flags": True` ?

Or do you think it should always have a default in the case of `metadata={"as_flags": True}`

Reply:
Good point! We need to keep that test case. I think it is a reasonable, though uncommon, use case to make a boolean field required.